addons/binding/org.openhab.binding.heos/ESH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heos" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+    
+   <channel-type id="control">
+       <item-type>Player</item-type>
+       <label>Control</label>
+       <description>Heos Control</description>
+    </channel-type>
+    <channel-type id="volume">
+        <item-type>Dimmer</item-type>
+        <label>Volume</label>    
+    </channel-type>
+    <channel-type id="mute">
+        <item-type>Switch</item-type>
+        <label>Mute</label>
+     </channel-type>
+     <channel-type id="group">
+        <item-type>Switch</item-type>
+        <label>Connect</label>
+     </channel-type>
+     <channel-type id="ungroup">
+        <item-type>Switch</item-type>
+        <label>Group</label>
+     </channel-type>
+     <channel-type id="title">
+        <item-type>String </item-type>
+        <label>Song</label>
+     </channel-type>
+     <channel-type id="interpret">
+        <item-type>String</item-type>
+        <label>Artist</label>
+     </channel-type>
+     <channel-type id="album">
+        <item-type>String</item-type>
+        <label>Album</label>
+     </channel-type>
+     <channel-type id="imageUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>Cover</label>
+     </channel-type>
+     <channel-type id="player">
+        <item-type>Switch</item-type>
+        <label>Player</label>
```

Comment:
I think its an old channel which was used at the beginning for adding player channel to the bridge. But then I changed to add and remove them dynamically. So I will test if it can be removed. Same for "group" and "favorite" channel.